index.ts

```diff
@@ -9,32 +9,66 @@ import { Sandbox } from '@e2b/sdk'
 import { onLog } from './log'
 import { cloneRepo, loginWithGH } from './gh'
 import { sleep } from './sleep'
-import { listFiles, makeCommit, makeDir, makePullRequest, readFile, saveCodeToFile } from './actions'
+import { listFiles, makeCommit, makeDir, makePullRequest, readFile, saveCodeToFile, runCode, runPowershellCommand } from './actions'
 
 const openai = new OpenAI()
 
 const AI_ASSISTANT_ID = process.env.AI_ASSISTANT_ID!
 
-async function initChat(): Promise<{ repoName: string; task: string }> {
-  let { repoName } = (await prompts({
-    type: 'text',
-    name: 'repoName',
-    message: 'Enter repo name (eg: username/repo):',
-  } as any)) as any
-
-	// Replace any backslashes in the repo name with forward slashes
-	repoName = repoName.replace(/\\/g, '/');
-
-  const { task } = (await prompts({
-    type: 'text',
-    name: 'task',
-    message: 'Enter the task you want the AI developer to work on:',
-  } as any)) as any
+async function initChat(): Promise<{ repoName: string; task: string; mode: string }> {
+  let { mode } = (await prompts({
+    type: 'select',
+    name: 'mode',
+    message: 'Choose mode:',
+    choices: [
+      { title: 'Just chit-chat and run some code or commands', value: 'chit-chat' },
+      { title: 'Code an already available repo', value: 'repo' },
+    ],
+  })) as any
+
+  let repoName = '';
+  let task = '';
+
+  if (mode === 'repo') {
+    let { repoName } = (await prompts({
+      type: 'text',
+      name: 'repoName',
+      message: 'Enter repo name (eg: username/repo):',
+    } as any)) as any
+  
+    // Replace any backslashes in the repo name with forward slashes
+    repoName = repoName.replace(/\\/g, '/');
+  
+    let { task } = (await prompts({
+      type: 'text',
+      name: 'task',
+      message: 'Enter the task you want the AI developer to work on:',
+    } as any)) as any
+  }
 
-  return { repoName, task }
+  return { repoName, task, mode }
+}  
+
+function createThread(repoURL: string, task: string, mode: string) {
+  let content;
+  if (mode === 'repo') {
+    content = `Pull this repo: '${repoURL}'. Then carefully plan this task and start working on it: ${task}`;
+  } else if (mode === 'chit-chat') {
```

Comment:
What's this good for? I would prefer to keep this simple and on the point, then introducing feature which won't be used